polaris/ocean/viz/transect/plot.py

```diff
@@ -0,0 +1,204 @@
+import cmocean  # noqa: F401
+import matplotlib.pyplot as plt
+import numpy as np
+
+from polaris.ocean.viz.transect.vert import (
+    interp_mpas_to_transect_cells,
+    interp_mpas_to_transect_nodes,
+)
+from polaris.viz.style import use_mplstyle
+
+
+def plot_transect(ds_transect, mpas_field=None, out_filename=None, ax=None,
+                  title=None, vmin=None, vmax=None, colorbar_label=None,
+                  cmap=None, figsize=(12, 6), dpi=200, method='flat',
+                  outline_color='black', ssh_color=None, seafloor_color=None,
+                  interface_color=None, cell_boundary_color=None,
+                  linewidth=1.0, transect_start='red', transect_end='green'):
```

Comment:
Okay, I'll add another bool parameter `plot_start_end` that defaults to `False`.  I don't like the idea of the colors defaulting to `None` because then we have to get them right every time so they match the start and end of the transect plot on the horizontal field.

Reply:
Fixed!